src/servers/main.cc

```diff
@@ -505,6 +516,69 @@ ParseFloatOption(const std::string arg)
   return std::stof(arg);
 }
 
+struct VgpuOption {
+  int gpu_device_;
+  int num_vgpus_;
+  uint64_t mem_limit_mbytes_;
```

Comment:
While `mem_limit_mbytes_` is `uint64_t`, you later use a `float` for the memory limit in mb: `std::map<int, std::vector<float>>& memory_limit_mb`. I might be missing something here but is there a reason for using different datatypes?

Reply:
The tensorflow API requires a float, but we're restricting the use to uint64_t.

Reply:
Understood